controllers/feeds/getOneFeedChat.js

```diff
@@ -69,7 +70,7 @@ const getOneFeedChatService = async (req, res) => {
     if (moment(data.results[0].expired_at).isBefore(moment().utc())) {
       return ErrorResponse.e400(res, 'Feed alredy expired');
     }
-    const {comments, upvotes, downvotes} = getReaction(
+    let {comments, upvotes, downvotes} = getReaction(
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Changing `const` to `let` for `{comments, upvotes, downvotes}` might introduce potential bugs if these variables are accidentally reassigned later in the code. If there's no need to reassign these variables, it would be safer to keep them as `const`.
```diff
- let {comments, upvotes, downvotes} = getReaction(
+ const {comments, upvotes, downvotes} = getReaction(
       req,
       data.results[0].latest_reactions,
       data.results[0].actor
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->